crates/alpen-ee/block-assembly/src/block.rs

```diff
@@ -0,0 +1,74 @@
+use std::num::NonZero;
+
+use alpen_ee_common::PayloadBuilderEngine;
+use alpen_reth_node::AlpenBuiltPayload;
+use strata_acct_types::{AccountId, Hash};
+use strata_ee_acct_runtime::{apply_final_update_changes, apply_input_messages};
+use strata_ee_acct_types::EeAccountState;
+use strata_ee_chain_types::ExecBlockPackage;
+use strata_snark_acct_types::MessageEntry;
+
+use crate::{package::build_block_package, payload::build_exec_payload};
+
+#[derive(Debug)]
+pub struct BlockAssemblyInputs {
+    /// EeAccountState of last block.
+    pub account_state: EeAccountState,
+    /// New inbox messages to be included in this block.
+    /// Can be empty.
+    pub inbox_messages: Vec<MessageEntry>,
+    /// Exec blkid of previous block.
+    pub parent_exec_blkid: Hash,
+    /// Timestamp of next block to be built in ms.
+    pub timestamp_ms: u64,
```

Comment:
You already specified in the docstring that is in milliseconds.
```suggestion
    pub timestamp: u64,
```

Reply:
wait, you're blocking merge on this? I'm not sure that we have this privilege